lib/server/server.js

```diff
@@ -170,7 +170,16 @@ addDataSources({
   sourcePath: servicePath
 })
 
-const govukFrontendVersion = require(path.join(serviceNodeModules, 'govuk-frontend', 'package.json')).version
+const govukFrontentPath = path.join(serviceNodeModules, 'govuk-frontend')
+// Patch govukHint to allow block-level elements
```

Comment:
Are there any other options here? Would be good for me to better understand this problem?

Reply:
We'll probably raise a PR on the govuk-frontend to resolve the issue of the hint being a span (ie. non block-level element) when we're sure how to handle this